README.md

```diff
@@ -584,6 +584,12 @@ To get your Public and Private API Key, follow these [instructions](https://www.
 To get the group ID and the app ID, the easiest way is to navigate to your Atlas Service App dashboard and copy from the URL.
 The URL format is https://realm.mongodb.com/groups/[groupId]/apps/[appId]/dashboard
 
+In addition to that, this value should be added to the `.env` file:
```

Comment:
Maybe specify that this is for development only?

Reply:
Actually, after the refactor to move the sendEmail code to a function, this `SENDGRID_API_KEY` value is not needed